typescript/web/src/components/website/Video/movie.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Heading, Flex, Center, AspectRatio } from "@chakra-ui/react";
+import * as React from "react";
+
+export const Movie = () => {
```

Comment:
As explained in the other file. This file is duplicate with `components/website/Movie/movie.tsx`.
Please merge them in a `IntroVideo` component located in `components/website/intro-video.tsx`